### ISSUE TITLE
Continue download even when private channel found

### ISSUE THREAD

**Issue by juanvelascogomez** [expectocode/telegram-export#93]
I've detected that when a client's information is being backed up and Telegram Exports detects a private Telegram channel while downloading, it just stops working. 

To fix it, you must search for the ID of the entity in question (private channel) and add it to the blacklist in the config file. 

Would it be a good idea to add a python "pass" command when Telegram Export detects a private channel while downloading the information so it continues downloading other groups without stopping?

**Comment by expectocode** [comment]
Good catch, thanks. Do you get an error message in this situation or does it just stop?

**Comment by juanvelascogomez** [comment]
I think you get an error message but not sure at the moment. I will post that message next time!

**Comment by juanvelascogomez** [comment]
Found it! 

![telethonerror](https://user-images.githubusercontent.com/7736733/48840113-0207ba80-ed8e-11e8-9a54-405927164b2a.PNG)

**Comment by juanvelascogomez** [comment]
Any plan on fixing this error? Thanks!

**Comment by Lonami** [comment]
Well as a bug, obviously there are plans to fix it. Who and when will fix it is another whole story, since there are things more important than this project.
